### PR TITLE
adding the option to open a file in new browser tab

### PR DESCRIPTION
**Pull request by AdarshdeepCheema**
Signed-off-by: Adarshdeep Cheema <adarshdeep.cheema@ibm.com>

### CODE REVIEW

**Review by DivergentEuropeans — CHANGES_REQUESTED** [review]
looks good to me, I'm going to double-up on what I said previously just a bit to make it make more sense. Because it's possible for the FT to exist in an app, but no Editor app to be installed, this right click option will do nothing. Looks like you chose to not check for existence of Editor which is okay, so let's instead rename the text to "Request Open in New Browser Tab" to let the user know, it's not up to the FT to accomplish this task

**Comment by AdarshdeepCheema** [comment]
changed the name of the text and skipped checking for the editor as discussed 
"Don't check for existence of Editor, that's a bad idea
Because Editor may not always be the only or best app to open a file in a new browser tab from FT"

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ZLUX Angular File Tree
 
 ## 0.11.0
-
+* Added the option to open a file in new browser tab
```
Comment (DivergentEuropeans) [inline CHANGELOG.md:4]: I think this was last release' version of the FT. Can you please make it 0.12, update package.json, and tag a new release to be used in the Editor?

**Review by DivergentEuropeans — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AdarshdeepCheema) [reply CHANGELOG.md:4]: Done

**Comment by AdarshdeepCheema** [comment]
working on making the package.json to use static dependencies

**Comment by AdarshdeepCheema** [comment]
merged via https://github.com/zowe/zlux-file-explorer/pull/141

**Pull request closed without merging**